packages/react/.eslintrc

```diff
@@ -104,14 +97,12 @@
       "files": [
         "**/*-fixture.{js,jsx}",
         "**/*.test.{js,jsx}",
+        "**/*.test.e2e.{js,jsx}",
```

Comment:
The overrides in eslintrc takes an array of objects, so you can split off cypress specific file overrides into their own object, too. That'll get rid of needing to add comments to turn off specific rules.

```js
// in .estlintrc overrides
"overrides": [
    {
      // ...
    },
    {
      "files": ["**/*.test.e2e.{js,jsx}"],
      "rules": {
        "jest/expect-expect": "off",
        "testing-library/prefer-screen-queries": "off"
      }
    }
```

Reply:
Thanks this was another to do to look into. I will make this change now.